test/command/suite/sharding/logical_range_filter/filter/match/nested_reference_vector/select.expected

```diff
@@ -98,7 +98,44 @@ logical_range_filter Logs timestamp   --limit -1   --filter 'users.name @ "ce"'
     ]
   ]
 ]
+#|-| [obj][open] <266>(<Logs_20150205.timestamp>):<64>(<column:fix_size>)
+#|-| [obj][open] <265>(<Logs_20150205>):<51>(<table:no_key>)
+#|-| [obj][open] <269>(<Timestamp_20150205>):<49>(<table:pat_key>)
+#|-| [obj][open] <270>(<Timestamp_20150205.index>):<72>(<column:index>)
 #|d| [logical_range_filter][select] <Logs_20150205>: limit is negative: <-1>
+#|-| [obj][open] <268>(<Logs_20150205.users>):<65>(<column:var_size>)
+#|-| [obj][open] <263>(<Users>):<49>(<table:pat_key>)
+#|-| [obj][open] <264>(<Users.name>):<65>(<column:var_size>)
+#|-| [obj][close] <263>(<Users>):<49>(<table:pat_key>)
+#|-| [obj][open] <263>(<Users>):<49>(<table:pat_key>)
+#|-| [obj][open] <271>(<Users.index_20150205>):<72>(<column:index>)
+#|-| [obj][open] <262>(<Terms>):<49>(<table:pat_key>)
+#|-| [obj][open] <272>(<Terms.users_name>):<72>(<column:index>)
+#|-| [obj][close] <271>(<Users.index_20150205>):<72>(<column:index>)
+#|-| [obj][close] <263>(<Users>):<49>(<table:pat_key>)
+#|-| [obj][close] <272>(<Terms.users_name>):<72>(<column:index>)
+#|-| [obj][close] <262>(<Terms>):<49>(<table:pat_key>)
+#|-| [obj][open] <262>(<Terms>):<49>(<table:pat_key>)
+#|-| [obj][open] <272>(<Terms.users_name>):<72>(<column:index>)
+#|-| [obj][open] <263>(<Users>):<49>(<table:pat_key>)
```

Comment:
This isn't closed.
Is this closed by #1333 and #1334, right?

Reply:
I have rebased onto `master` and updated tests and confirmed it is closed.
However, the `Users` table repeat `open` and `close`.
I'm little concern about it in term of performance...